tests/common/experimental/test_reducers_and_aggregators.py

```diff
@@ -297,15 +296,16 @@ def test_noop_aggregator(self):
         for val in aggregated:
             assert fns.allclose(val, self.get_nncf_tensor(input_))
 
-    def test_shape_aggregator(self):
-        aggregator = ShapeAggregator()
+    def test_noop_aggregator_return_first(self):
+        aggregator = NoopAggregator(None, return_first=True)
```

Comment:
That's a really small change, but I suggest to add a small test case for it in this function